packages/gestalt/src/Typeahead.js

```diff
@@ -339,6 +303,8 @@ TypeaheadWithForwardRef.propTypes = {
     PropTypes.exact({
       label: PropTypes.string.isRequired,
       value: PropTypes.string.isRequired,
+      // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Deduping this type may obviate the need for this suppression as well

Reply:
Sadly didn't seem to help